lxd/migrate.go

```diff
@@ -245,3 +245,54 @@ func (s *migrationSink) Connect(op *operations.Operation, r *http.Request, w htt
 	// operation actually exists.
 	return api.StatusErrorf(http.StatusForbidden, "Invalid migration sink secret")
 }
+
+type conversionSink struct {
```

Comment:
Can we keep these types together with the functions in migrate_instance.go?

Or put them together in a new file called convert_instance.go perhaps?

Reply:
I've combined everything in `convert_instance.go`, as suggested.